app/views/publishers/vacancies/applications.html.slim

```diff
@@ -0,0 +1,72 @@
+- content_for :page_title_prefix, current_organisation.name
+
+= render BannerComponent.new do
+  = govuk_breadcrumbs breadcrumbs: { "#{t("breadcrumbs.active_jobs")}": root_path, "#{@vacancy.job_title}": root_path }
```

Comment:
Should both these breadcrumbs really point to the same path?

Reply:
no, good spot and i havent tested it either

Reply:
in fact the second shouldnt be a link at all

Reply:
according to figma anyway, ill run it past chris